kyaml/fn/runtime/runtimeutil/functiontypes.go

```diff
@@ -33,6 +34,78 @@ const (
 	UserNobody ContainerUser = "nobody"
 )
 
+const defaultEnvValue string = "true"
+
+// ContainerEnvs contains the environment variables for the container
```

Comment:
It's _one_ environment, with multiple environment variables, flags, etc. defined inside.
suggest:
```
// ContainerEnv defines the environment present in a container.
```

Reply:
What i mean here - i don't think the struct's name should have an s on the end.